src/engage/enrollments/enrollment.search.ts

```diff
@@ -0,0 +1,48 @@
+import {ZapierItem} from '../../types'
+import {enrollmentSearches, enrollmentTriggers} from '../keys'
+import EnrollmentResource from './enrollment.resource'
+import {searchPrefixedField} from '../../utils/fieldsHelpers'
+import {searchActionDetails} from '../../utils/operations'
+import {searchEnrollmentsByCriteria} from '../common'
+import {leadSearches, leadTriggers} from '../../lead/keys'
+
+export const EnrollmentSearch: ZapierItem = {
+    key: enrollmentSearches.enrollmentSearch,
+    noun: 'Enrollment',
+    display: {
+        label: 'Find Enrollment in Catalog',
+        description: 'Finds a enrollment by Enrollment ID and resource ID. Requires Reach subscription.'
```

Comment:
@piotrkpaul let me know how this changes after we talked on Tuesday - I believe this should be find enrolment by sequence id/name and lead id.

Then a copy should be Finds a sequence enrolment for a given lead by sequence name or sequence ID.